microsoft-edge/webview2/how-to/set-preview-channel.md

```diff
@@ -6,259 +6,410 @@ ms.author: msedgedevrel
 
    ![Using the Windows search bar to find where to edit environment variables](./set-preview-channel-images/search-bar-edit-sys-env-vars.png)
 
-1. In the **System Properties** dialog, select the **Advanced** tab, and then select the **Environment Variables** button.
+   The **System Properties** dialog opens.
+
+1. Select the **Advanced** tab, and then click the **Environment Variables** button:
 
    ![The Environment Variables button in the System Properties dialog](./set-preview-channel-images/system-properties-env-vars.png)
 
-1. In the **User variables** section of the **Environment Variables** dialog, select **New**.
+   The **Environment Variables** dialog opens.
+
+1. In the **User variables** section at the top, click the **New** button.  The **New User Variable** dialog opens.
 
-1. In the **New User Variable** dialog, set the **Variable name** to `WEBVIEW2_BROWSER_EXECUTABLE_FOLDER`, and set the **Variable value** to the path to your preferred browser channel or fixed version binaries.
+1. In the **Variable name** text box, enter the environment variable name, either `WEBVIEW2_CHANNEL_SEARCH_KIND` or `WEBVIEW2_BROWSER_EXECUTABLE_FOLDER`.
 
-   Alternatively, set the **Variable name** to `WEBVIEW2_RELEASE_CHANNEL_PREFERENCE`, and set the **Variable value** to `1` to reverse the search order, to use preview channels (Beta, Dev, or Canary) first. Any value besides `1` indicates the default search order.
+1. In the **Variable value** text box, do either of the following:
 
-1. Click **OK** to close the dialogs.
+   * If setting the `ChannelSearchKind`, enter `1` to reverse the search order from the default, to use the preview channels first (Canary, Beta, then Dev), followed by the WebView2 Runtime.  Any value other than `1` causes the default search order to be used (WebView2 Runtime, Edge Dev, Edge Beta, and then Edge Canary).
 
-   ![Adding a new environment variable, as a user variable](./set-preview-channel-images/env-vars-new-user-variable.png)
+   * If setting the `BrowserExecutableFolder`, enter the path to your preferred browser channel or fixed-version binaries.
 
-### Which app is affected
+1. Click the **OK** buttons to close the dialogs.
 
-The above approach sets the environment variable for all WebView2 apps, not just the app you're testing.  To set this environment variable for only the WebView2 app which you are testing, if you're running your app from the command prompt, set the environment variable `WEBVIEW2_RELEASE_CHANNEL_PREFERENCE=1`.  That sets the environment variable just for the current `cmd.exe` command-prompt process and for any new child processes from that `cmd.exe` instance.  Then the environment variable only applies to the WebView2 app that you're testing.
 
-If you use the `WEBVIEW2_RELEASE_CHANNEL_PREFERENCE` environment variable, you can set it to the following values.
+<!-- ------------------------------ -->
+#### How to deploy preview channels
```

Comment:
```suggestion
#### How do I deploy Microsoft Edge preview channels?
```

Reply:
Or maybe this doesn't need to be in FAQ

Reply:
FAQs lack structure; prefer structured docs rather than FAQ format.

Reply:
Dissolved FAQ; promoted and standardized its child headings.